### PR TITLE
remove Unused parameter 'sf'  in calcTrendValue function

### PR DESCRIPTION
**Pull request by liguozhong**
remove Unused parameter 'sf'  in calcTrendValue function

### CODE REVIEW

File: promql/functions.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func instantValue(vals []parser.Value, out Vector, isRate bool) Vector {
 // The argument "s" is the set of computed smoothed values.
```
Comment (brian-brazil) [inline promql/functions.go]: Would you mind fixing this comment while you're here?

Comment (liguozhong) [reply promql/functions.go]: ok , done.

File: promql/functions.go [hunk 2]
```diff
@@ -193,9 +193,9 @@ func instantValue(vals []parser.Value, out Vector, isRate bool) Vector {
 
 // Calculate the trend value at the given index i in raw data d.
 // This is somewhat analogous to the slope of the trend at the given index.
-// The argument "s" is the set of computed smoothed values.
-// The argument "b" is the set of computed trend factors.
-// The argument "d" is the set of raw input values.
+// The argument "s0" is the set of computed smoothed values.
+// The argument "s1" is the set of computed trend factors.
+// The argument "b" is the set of raw input values.
```
Comment (roidelapluie) [inline promql/functions.go]: what is `tf`?

Comment (liguozhong) [reply promql/functions.go]: done. Thanks for your review. 'tf' 's comment have been added

File: promql/functions.go (hunk 1 above)
Comment (brian-brazil) [reply promql/functions.go]: Thanks, none of these are sets anymore.

Comment (liguozhong) [reply promql/functions.go]: ok，done

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**